Rename the ES preset to ESX and create a proper preset file for the ES rim

plumpo opened the pull request:
Fixes the key mappings to represent the buttons on the Moza ES wheel. The ones that came with the game are mapped for the ESX Xbox wheel.

plumpo committed:
Add files via upload

Fixes the key mappings to represent the buttons on the Moza ES wheel. The ones that came with the game are mapped for the ESX Xbox wheel.

Lawstorant reviewed (changes requested):
Please add the stock esx preset and name it accordingly :)

plumpo committed:
Add files via upload

Just the original ES wheel configuration, but renamed to ESX to represent the actual bindings for that wheel. The name will show up in game as lng_moza_rim_esx.

plumpo commented:
Done

> Please add the stock esx preset and name it accordingly :)
